data/starts.txt

```diff
@@ -0,0 +1,199 @@
+	`	To your disappointment, the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life.`
+	`	You've heard of new captains stripping down their ships to recoup the initial buy-in, which is crossing your mind as you see the financial burden you're about to take on. While waiting for paperwork to be processed, you scan over an Aphid pamphlet in the office. The pricey shield regenerator could be traded in if dangerous areas are avoided... and perhaps even the anti-missile system could be downgraded or outright removed... one of the Sand Cells could be sold back to make more space for cargo...`
+	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
+	name
+	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
+	choice
+		`	(Go to the shipyard.)`
+			decline
+		`	(Go see what the captain has to offer first.)`
+	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully he has not yet sold the Grasshopper.`
+	`	The captain greets you in Hai. You are embarrassed to have to explain to them that you never learned their language. Most Hai speak your language however, and, as usual, they are able to switch to speaking yours without much difficulty.`
+	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
+	choice
+		`	"Yes indeed."`
+		`	"How did you know?"`
+		`	"Sorry, not interested."`
+			decline
+	`	"I can see the will to explore in your eyes, but it is not only that: I know that some humans want to head back to their space, now that it is safer to do so. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
+	scene "thumbnail/hai grasshopper"
+	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
+	choice
+		`	"Sold!"`
+		`	"Sorry, not interested."`
+			decline
+	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. Because of how excited you are to finally fly, you miss everything other than repeated reference to the 'exceptional' reinforced armor.`
```

Comment:
> Because of how excited you are to finally fly, you miss everything other than repeated reference to the 'exceptional' reinforced armor.

That feels a little contrived. How about this:
```suggestion
	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`
```